R/pagination.R

```diff
@@ -1290,7 +1299,7 @@ paginate_to_mpfs <- function(obj,
       rep_cols = rep_cols,
       page_num = page_num
     )
-    return(unlist(deep_pag, recursive = FALSE))
+    return(deep_pag)
```

Comment:
@Melkiades did you test this with multiple page_bys? I'm pretty sure this breaks existing behavior

Reply:
Yes! it works fine, also with page numbers

Reply:
You're right. I had thought that do_forced_pag was acting recursively but its not; one call is expected to do *all* of the forced pagination, e.g.:


```
> formatters:::do_forced_paginate(tbl)
$F.A

SEX: F
STRATA1: A

———————————————————————————————————————————————————————————————————————————————————
                                            A: Drug X   B: Placebo   C: Combination
———————————————————————————————————————————————————————————————————————————————————
ASIAN                                                                              
  Mean                                        29.00       31.07          33.73     
BLACK OR AFRICAN AMERICAN                                                          
  Mean                                        32.60       32.80          39.33     
WHITE                                                                              
  Mean                                        34.40       33.67          29.67     
AMERICAN INDIAN OR ALASKA NATIVE                                                   
  Mean                                         NA         35.00          38.00     
MULTIPLE                                                                           
  Mean                                         NA           NA             NA      
NATIVE HAWAIIAN OR OTHER PACIFIC ISLANDER                                          
  Mean                                         NA           NA             NA      
OTHER                                                                              
  Mean                                         NA           NA             NA      
UNKNOWN                                                                            
  Mean                                         NA           NA             NA      

$F.B

SEX: F
STRATA1: B

———————————————————————————————————————————————————————————————————————————————————
                                            A: Drug X   B: Placebo   C: Combination
———————————————————————————————————————————————————————————————————————————————————
ASIAN                                                                              
  Mean                                        29.55       38.73          41.45     
BLACK OR AFRICAN AMERICAN                                                          
  Mean                                        34.33       25.67          27.25     
WHITE                                                                              
  Mean                                        35.00       32.25          33.40     
AMERICAN INDIAN OR ALASKA NATIVE                                                   
  Mean                                        38.33       38.00          36.00     
MULTIPLE                                                                           
  Mean                                         NA           NA             NA      
NATIVE HAWAIIAN OR OTHER PACIFIC ISLANDER                                          
  Mean                                         NA           NA             NA      
OTHER                                                                              
  Mean                                         NA           NA             NA      
UNKNOWN                                                                            
  Mean                                         NA           NA             NA      



```